tcms/core/middleware.py

```diff
@@ -32,3 +34,20 @@ def process_request(self, request):
                       }
                 )
             )
+
+
+class CheckUnappliedMigrationsMiddleware(MiddlewareMixin):
+    def process_request(self, request):
+        executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
+        plan = executor.migration_plan(executor.loader.graph.leaf_nodes())
+        if plan:
+            apps_waiting_migration = sorted({migration.app_label for migration, backwards in plan})
```

Comment:
this also prints I have 57 unapplied migrrations when all of them are actually applied. See how `showmigrations` works.

Reply:
Ahh, didn't notice that, will switch to `MigrationLoader` from `showmigrations`.